app/src/main/java/helium314/keyboard/latin/spellcheck/AndroidWordLevelSpellCheckerSession.java

```diff
@@ -171,58 +205,31 @@ public void onClose() {
     }
 
     private static final int CHECKABILITY_CHECKABLE = 0;
-    private static final int CHECKABILITY_TOO_MANY_NON_LETTERS = 1;
```

Comment:
Why is this removed?

Reply:
I would like to be notified if I wrote "My phone number is0123456789" <- missed a space after "is"

Reply:
The problem I see here is that this might not be wanted by some people (it might have been introduced for a reason), and with the current style of PR I can't just simply revert this change if it turns out clearly unpopular.